<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="p_lookbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>400</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>상품 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iv_lookbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>150</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>재고 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="product_addbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>350</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>상품 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="product_changebtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>300</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>상품 수정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pcancelbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>300</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>지정취소</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>39</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="acancelbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>350</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전체취소</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paybtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>401</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>결제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="product_deletebtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>250</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>상품 삭제</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>481</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="salebtn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>판매</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lookbtn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lookbtn_3">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>폐기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lookbtn_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>정산</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitbtn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>종료</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="salelist_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>판매목록</string>
    </property>
   </widget>
   <widget class="QLabel" name="basketlist_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>장바구니</string>
    </property>
   </widget>
   <widget class="QLabel" name="cartegpry_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>3</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>카테고리</string>
    </property>
   </widget>
   <widget class="QLabel" name="payhistory_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>결제내역</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="product_table">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>39</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>600</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>55</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Qt
POS Program</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="basketlist_table">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inventory_addbtn">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>100</y>
      <width>100</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>발주</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="iv_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>9</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="p_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>9</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="pay_table">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>39</y>
      <width>461</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitbtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>722</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
